README.md

```diff
@@ -8,16 +8,12 @@ https://bitcoincore.org/en/download/.
 
 Further information about Bitcoin Core is available in the [doc folder](/doc).
 
-What is Bitcoin?
-----------------
+What is Bitcoin Core?
+---------------------
 
-Bitcoin is an experimental digital currency that enables instant payments to
-anyone, anywhere in the world. Bitcoin uses peer-to-peer technology to operate
-with no central authority: managing transactions and issuing money are carried
-out collectively by the network. Bitcoin Core is the name of open source
-software which enables the use of this currency.
-
-For more information read the original Bitcoin whitepaper.
+Bitcoin Core is a full node implementation for the Bitcoin P2P network. It also
+includes a wallet and Qt-based graphical user interface, that can be
+optionally built.
 
 License
 -------
```

Comment:
As you are removing "experimental", could clarify that the software doesn't come with any warranty and users are using it at their own risk? (According to the MIT license)

Reply:
"License" follows it, right? I also removed "open source".

Reply:
It just doesn't describe Bitcoin anymore. Bitcoin might be experimental it might not be experimental, determining that is out of scope of this project.

Reply:
Sure, no strong opinion either way. I just thought it couldn't hurt to explain the warranty part to users that do not read the license (who does that?)

Reply:
We could paste the entire MIT license into the License part. It's a few lines.

Reply:
I think one of the reasons to keep "experimental" was to make users more cautious and less yolo. I think caution still applies (and probably will forever).